pipewire/pipewire-jack-runtime.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "pipewire-jack-runtime",
+    "buildsystem": "simple",
+    "build-commands": [
+        "install -Dm644 *.h -t /app/include/jack/",
+        "install -Dm644 pipewire-jack-runtime.pc /app/lib/pkgconfig/jack.pc"
```

Comment:
why is copying the pkg config file necessary ? This should be done automaitcally